### DISCUSSION TITLE
[Feature Request] macOS support?

### DISCUSSION THREAD

**Discussion by UInt2048** [legendary-gl/legendary#169] [Q&A] [answered]
It's possible to get and install a 64-bit version of Wine on macOS using the command `brew install wine-devel`, and it's ran using `wine64`.

Is there a specific reason that Legendary won't support macOS (cf. #12 and #28), even if just to run things that are 64-bit and Windows (a free example being [Rocket League](https://www.epicgames.com/store/en-US/product/rocket-league/home))? Is there anything I could do to help?

**Comment by derrod** [comment]
I have no Mac and no interest in supporting macOS. I'd imagine legendary might work as-is anyway since macOS is reasonably similarly structured to Linux? But I certainly don't want to deal with maintenance, issues, and testing on macOS.

**Comment by UInt2048** [comment, marked as answer]
I was able to get it to install a game. Here's how one could do it:

1. `brew install wine-devel` - this creates wine64
2. `ln -s /usr/local/bin/wine64 /usr/local/bin/wine` - symlinks it to wine for use by Legendary
3. `pip3 install legendary-gl` - requires Python 3 to be on the system; may need to use sudo

From here, steps are as in the README:

4. `legendary auth` - signs you in with Epic
5. `legendary list-games` - lists games
6. `legendary install Sugar` - installs a game using the app name in parentheses from step 5 (not user-facing name) to directory that it outputs at the beginning of this command (could be useful if needed to be launched directly)
7. Use Wine, or `legendary launch Sugar`, or whatever - this was just supposed to install a game...
